### PR TITLE
use milliseconds instead to support smaller boards

### PR DESCRIPTION
**Pull request by rhooper**
Addresses #62

### CODE REVIEW

File: adafruit_led_animation/animation/__init__.py [hunk 1]
```diff
@@ -201,11 +201,11 @@ def speed(self):
         """
         The animation speed in fractional seconds.
         """
-        return self._speed_ns / NANOS_PER_SECOND
+        return self._speed_ns / MS_PER_SECOND
```
Comment (caternuson) [inline adafruit_led_animation/animation/__init__.py]: Change `self._speed_ns` to `self._speed_ms`? Cosmetic, but could help prevent future confusion.

**Comment by caternuson** [comment]
Looks good. One suggested cosmetic code tweak. Sort of a deja vu feeling - I did pretty much these same mods yesterday, made this easy to review :)

Tested on Trinket M0 with a basic blink, and it works.

**Comment by rhooper** [comment]
Hmm - Tested with 6.x and get `No long integer support`

**Comment by rhooper** [comment]
@caternuson can you look again?

**Comment by caternuson** [comment]
I tested with 5.3.1. But seeing same with 6.0 alpha 2. I'm guessing this a bug in current 6.x build. I've asked the devs. Stay tuned....
```python
Adafruit CircuitPython 6.0.0-alpha.2 on 2020-07-23; Adafruit Trinket M0 with samd21e18
>>> import time
>>> dir(time)
['__name__', 'localtime', 'mktime', 'monotonic', 'monotonic_ns', 'sleep', 'struct_time', 'time']
>>> time.monotonic()
90.376
>>> time.monotonic_ns()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NotImplementedError: No long integer support
>>> 
```

**Comment by caternuson** [comment]
So...sounds like it's a behavior change for 6.x. Instead of not having the methods present, they will be present but throw an exception.

So I guess the import logic needs to change accordingly.

**Comment by caternuson** [comment]
Related:
https://github.com/adafruit/circuitpython/issues/2812

**Comment by rhooper** [comment]
The latest version in this branch should work on Trinket M0 on 5.3.1 and 6.0.0-latest (6.0.0-alpha.0-556-g3197c579e-dirty)

**Comment by caternuson** [comment]
@rhooper OK, grabbed latest. Yep, works now.

**Comment by rhooper** [comment]
@caternuson  Looking for an approval to merge

**Review by caternuson — APPROVED** [review]

**Pull request merged**